digdag-server/src/main/java/io/digdag/server/ServerConfig.java

```diff
@@ -32,6 +32,8 @@
 
     public Optional<String> getServerRuntimeInfoPath();
 
+    public Optional<String> getUiConfigJsPath();
```

Comment:
It seems we also need to add as a new parameter in these lines.
https://github.com/treasure-data/digdag/blob/935e402584e627660e12348ff32453e77da5373a/digdag-server/src/main/java/io/digdag/server/ServerConfig.java#L76-L96

How about this...? This is just a suggestion.

```java
            .uiConfigJsPath(config.getOptional("server.ui.config-js-path", String.class))
```